static/js/core.js

```diff
@@ -9,15 +9,23 @@ function alpineCore() {
         version: '0.0.0',
         isFirstVisit: false,
         scrollTop: window.scrollY,
+        newestVersion: '0.0.0',
+        newVersionAvailable: false,
 
         initPage() {
             window.onscroll = () => {
                 this.scrollTop = window.scrollY;
             };
 
-            apiV2('GET', '/api/v2/health').then((response) => {
+            apiV2('GET', 'https://api.github.com/repos/mitre/caldera/releases').then((response) => {
```

Comment:
Moved the fetch command to here to it get the version async, that way when we do the comparison we can be sure we have the value first.